packages/mui-base/src/Select/Select.test.tsx

```diff
@@ -660,11 +660,32 @@ describe('<Select />', () => {
       });
 
       expect(handleChange.callCount).to.equal(1);
-      expect(handleChange.args[0][0]).to.haveOwnProperty('type', 'click');
-      expect(handleChange.args[0][0]).to.haveOwnProperty('target', optionTwo);
```

Comment:
`target` value in `clonedEvent` is overriden by {value: newValue} which is coming from here. 

Same thing is happening in material-ui select as well. in the below demo, click on one of the option, you can see in the console that only `value and name` are printed.

https://stackblitz.com/edit/react-elrafy?file=demo.tsx